sdk/search/azure-search-documents/azure/search/documents/_service/_search_service_client.py

```diff
@@ -0,0 +1,82 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from typing import TYPE_CHECKING
+
+from azure.core.pipeline.policies import HeadersPolicy
+from azure.core.tracing.decorator import distributed_trace
+from ._generated import SearchServiceClient as _SearchServiceClient
+from .._version import VERSION
+
+if TYPE_CHECKING:
+    # pylint:disable=unused-import,ungrouped-imports
+    from typing import Any, Union
+    from .. import SearchApiKeyCredential
+
+
+class SearchServiceClient(object):
+    """A client to interact with an existing Azure search service.
+
+    :param endpoint: The URL endpoint of an Azure search service
+    :type endpoint: str
+    :param credential: A credential to authorize search client requests
+    :type credential: SearchApiKeyCredential
+
+    .. admonition:: Example:
+
+        .. literalinclude:: ../samples/sample_authentication.py
+            :start-after: [START create_search_service_client_with_key]
+            :end-before: [END create_search_service_client_with_key]
+            :language: python
+            :dedent: 4
+            :caption: Creating the SearchServiceClient with an API key.
+    """
+
+    def __init__(self, endpoint, credential, **kwargs):
+        # type: (str, SearchApiKeyCredential, **Any) -> None
+
+        headers_policy = HeadersPolicy(
+            {
+                "api-key": credential.api_key,
+                "Accept": "application/json;odata.metadata=none",
```

Comment:
@brjohnstmsft is this Accept header correct/appropriate/necessary for the service client as well?

Reply:
No, for the service client you'll need `application/json;odata.metadata=minimal`. Otherwise, `@odata.type` annotations won't be in the response. You need those for correct deserialization of polymorphic model types.

Reply:
👍  fixed in https://github.com/Azure/azure-sdk-for-python/pull/10371/commits/df761e9b662e0d395cf0ecc0b7c5508578d5e0f7